mods/ts/sequences/structures.yaml

```diff
@@ -846,6 +846,13 @@ proc: # TODO: unused narefn_a, narefn_b
 	make: ntrefnmk
 		Length: 20
 		ShadowStart: 20
+	dock: ntrefn_a
+		Frames: 5,4,3,2,1,1,2,3,4,5
```

Comment:
`WithDockingOverlay` isn't suitable for this animation.  It should play backwards once at the start of the dock, stay on the first frame for the duration of the dock, and then play forwards once at the end.